### ISSUE TITLE
Issue when importing the main less file

### ISSUE THREAD

**Issue by diosney** [zavoloklom/material-design-color-palette#3]
When doing:

``` less
@import '../vendors/bower_components/bootstrap/less/bootstrap.less';
@import '../vendors/bower_components/material-design-typography/less/material-design-typography.less';
```

is triggering:

```
>> SyntaxError: variable @mdc-#9e9e9e-500 is undefined in src/vendors/bower_components/bootstrap/less/type.less on line 128, column 3:
>> 127   color: #fff;
>> 128   .bg-variant(@brand-primary);
>> 129 }
```

**Comment by diosney** [comment]
I dug a little more and found out that this library is overwriting the `.bg-variant()` mixin, so they conflict each other.

**Comment by zavoloklom** [comment]
Fixed since [1.1.0](https://github.com/zavoloklom/material-design-color-palette/releases)
